SpaceX/ActivityIndicator/ActivityIndicator.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  ActivityIndicator.swift
+//  SpaceX
+//
+//  Created by Julia on 26.07.2022.
+//
+
+import UIKit
+
+class ActivityIndicator: UIImageView {
+    var rocket1 = UIImageView(image: UIImage(named: "rocket")?.withTintColor(.sxItemColor))
+    var rocket2 = UIImageView(image: UIImage(named: "rocket")?.withTintColor(.sxRed))
+    let positions: [CGRect] = [CGRect(x: 20, y: 10, width: 30, height: 30),
+                               CGRect(x: 30, y: 7.5, width: 40, height: 40),
+                               CGRect(x: 50, y: 10, width: 30, height: 30),
+                               CGRect(x: 30, y: 12.5, width: 20, height: 20)]
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configure()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func configure() {
+        translatesAutoresizingMaskIntoConstraints = false
+        rocket1.layer.zPosition = 2
+        self.addSubview(rocket1)
+        self.addSubview(rocket2)
+        rocket2.layer.zPosition = 1
+    }
```

Comment:
When the code is in UIImageView(This is UiView) we may not use `self` 
It will be just `addSubview(rocket1)` instead of `self.addSubview(rocket1)`